.github/PULL_REQUEST_TEMPLATE/nomad_pull_request_template.md

```diff
@@ -11,44 +11,68 @@ If your content is an update to:
   Choose **base: main** when you are updating published documentation, and you
   want your changes published when the PR is merged. We publish Nomad content
   from the `main` branch.
+
 - Upcoming Nomad release
 
   Choose the branch for the Nomad release that your content is for. Nomad
   release branches use the `nomad/<exact-release-number>` format. If you are not
   able to find the upcoming Nomad release branch that you are looking for,
   contact the tech writer that works with the Nomad team.
 
+- Upcoming Nomad release but not sure which one
+
+  - Choose the `main` branch.
+  - Add the "do not merge" label.
+  - Convert the PR to a DRAFT.
+  - Put an explanation in the **Description** section.
+
+  The tech writer coordinates with Nomad engineering and updates
+  the docs PR base branch when the code is slotted into an upcoming release.
+
 **Backports**
 
 This repo stores previous version docs in folders instead of branches. There are
 no backport labels.  If you backported your code PR to previous branches, update
 the docs content in the corresponding folders. For example, if the current
 release is 1.10.x and you backported your code to 1.9.x and 1.8.x, update the docs content
-in the v1.10.x, v1.9.x, and v1.8.x folders.
+in the v1.10.x, v1.9.x, and v1.8.x folders. If you can't find the relevant files in previous versions,
+add a note in the description so that the tech writer can update previous versions.
```

Comment:
```suggestion
in the v1.10.x, v1.9.x, and v1.8.x folders. If you can't find the relevant files in previous versions,
add a note to your PR description. The tech writer will help ensure that the previous versions
receive the correct updates.
```
In general, I like actionable commands that you're directing someone to do to end sentences.